cluster/src/main/scala/io/snappydata/gemxd/SparkSQLExecuteImpl.scala

```diff
@@ -470,6 +470,7 @@ object SnappySessionPerConnection {
     if (session != null) session
     else {
       val session = SnappyContext().snappySession
+      session.conf.set("snappydata.sql.planCaching", "true")
```

Comment:
Better to use Property.PlanCaching: Property.PlanCaching.set(session.sessionState.conf, true)

Reply:
ok